libs/ledger/include/ledger/chain/main_chain.hpp

```diff
@@ -44,6 +44,16 @@
 namespace fetch {
 namespace ledger {
 
+using std::int8_t;
```

Comment:
It might be non-standard but the rest of the code base doesn't using the std prefix. Lets ditch this and then we can cross the bridge when it doesn't work on a new platform in the future